client/extensions/woocommerce/app/settings/payments/payments-off-site.js

```diff
@@ -25,7 +52,17 @@ class SettingsPaymentsOffSite extends Component {
 							'information'
 						)
 					} />
-				<Card></Card>
+					<ListTable>
+						<ListHeader>
+							<ListRowField width="30%">Method</ListRowField>
```

Comment:
Could we pass in a className prop and thereby use CSS instead please? That way we can keep styling all in one place.